e2e/nextjs-routes.d.ts

```diff
@@ -1,11 +1,12 @@
 // THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.'
 // Run `yarn nextjs-routes` to regenerate this file.
 /* eslint-disable */
+// prettier-ignore
```

Comment:
Does this ignore the next node and not the whole file? https://prettier.io/docs/en/ignore.html#javascript

Reply:
`"nextjs-routes"` is the next node and the only dynamic piece that is likely to be formatted, just curious if there are semantics for a whole file ignore that I'm unaware of

Reply:
Oh you're right, that's unfortunate.

Reply:
Guess we just force users to ignore it?

Reply:
I wonder if [range ignore](https://prettier.io/docs/en/ignore.html#range-ignore) works in JS?

Reply:
Unfortunately not https://github.com/prettier/prettier/issues/5287

Reply:
That's unfortunate. We only have 3 nodes so I suppose we could add the ignore above each module declaration?

Reply:
done https://github.com/tatethurston/nextjs-routes/pull/27